### ISSUE TITLE
:IcedSourcePopupShow and :IcedSourceShow throw errors in some cases

### ISSUE THREAD

**Issue by Cnly** [liquidz/vim-iced#191]
In some cases, instead of showing the source, `:IcedSourcePopupShow` throws the following error:

```
Failed to open popup: 'Vim(return):E731: using Dictionary as a String'.
```

and `:IcedSourceShow` throws:

```
Vim(call):E119: Not enough arguments for function: <SNR>67_noop
function <SNR>68__worker[12]..<SNR>67__publish, line 8
```

Steps to reproduce:

1. Set up the minimal configuration as described at https://liquidz.github.io/vim-iced/#minimal_configuration
1. `lein new app test`
1. Open `src/test/core.clj` and run `:IcedJackIn`
1. Move the cursor to `defn` and run `:IcedSourcePopupShow` and `:IcedSourceShow`. The commands should both error.
1. Move the cursor to `-main` and run them again. The source should be shown properly.

Environment: Debian, NVIM `v0.5.0-404-g49cd750d6`, vim-iced revision `5764f80` (pulled just now)

**Comment by liquidz** [comment]
@Cnly Thanks for your reporting!
I can't take time today, so I'll have a look tomorrow.

**Comment by liquidz** [comment]
@Cnly I could reproduce the problem and fixed on `dev` branch.
Could you try `dev` branch?

I verified with `NVIM v0.5.0-416-g2b00d1d09`.

**Comment by Cnly** [comment]
Can confirm this is fixed on `dev` now. Thanks!

**Comment by liquidz** [comment]
Thanks! I'll cut a new release this weekend.

**Comment by liquidz** [comment]
Closed by #192
